### PR TITLE
Support editing messages

### PR DESCRIPTION
**Pull request by basine**
**In this PR:**

Added integration tests for editing messages.
Fixed behaviour of `ZMConversationTranscoder` that updated conversation properties when processing events. This was interfering with the updates done when converting message events into messages.

Needs
- [x] Release of ZMCDataModel with https://github.com/wireapp/wire-ios-data-model/pull/12

### CODE REVIEW

**Comment by marcoconti83** [comment]
wireapp/wire-ios-data-model#12 was merged

File: Tests/Source/Integration/ConversationTests+MessageEditing.m [hunk 1]
```diff
@@ -0,0 +1,372 @@
+//
+//  ConversationTests+MessageEditing.m
+//  zmessaging-cocoa
+//
+//  Created by Sabine Geithner on 11/08/16.
```
Comment (daehn) [inline Tests/Source/Integration/ConversationTests+MessageEditing.m]: The copyright header is missing here

**Comment by daehn** [comment]
I will resolve the conflicts 💣

**Comment by daehn** [comment]
I bumped `wire-ios-data-model` to the new version, so this should be ready to merge @basine ?

**Pull request merged**